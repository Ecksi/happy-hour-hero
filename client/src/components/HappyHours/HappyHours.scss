.happyHoursContainer .findLocationDropdown {
  display: block;
  margin: -20px auto 20px auto;
  width: 68%;
}

.happyHoursContainer .homeSearchSubmit {
  display: none;
}