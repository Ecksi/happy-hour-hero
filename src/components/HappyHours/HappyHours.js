import React, { Component } from 'react';
import './HappyHours.css';

class HappyHours extends Component {
  constructor (props) {
    super(props);
    
  }

  render() {
    return (
      <section className="happyHoursContainer">
        <h1>Happy Hours!</h1>
      </section>
    );
  }
}

export default HappyHours;
