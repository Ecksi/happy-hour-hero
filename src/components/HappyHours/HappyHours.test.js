import React from 'react';
import { shallow } from 'enzyme';
import { HappyHours } from './HappyHours';

describe('HappyHours', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HappyHours address="yo mama" />);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});