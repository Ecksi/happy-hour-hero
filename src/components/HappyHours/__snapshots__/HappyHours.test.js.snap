// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HappyHours matches the snapshot 1`] = `undefined`;
